README.md

```diff
@@ -409,14 +405,9 @@ component.
 Will install the default `infinity-loader` template into your host app, at
 `app/templates/components/infinity-loader`.
 
-* **scrollable**
+* **[DEPRECATED] scrollable** 
 
-```hbs
-{{infinity-loader scrollable="#content"}}
-```
```

Comment:
This may not be such a good idea.  If the scrollable area isn't the window (overflow: scroll, height: 200px), then the infinity load trigger will happen wayyyyy to early.